test/homework_01/test_calc.py

```diff
@@ -0,0 +1,16 @@
+from homework_01.calc import check_power_of_2
+
+
+def test_positive_case():
+    """Testing that actual powers of 2 give True"""
+    assert check_power_of_2(65536)
+
+
+def test_negative_case():
+    """Testing that non-powers of 2 give False"""
+    assert not check_power_of_2(12)
+
+
+def test_zero():
+    """Testing that 0 gives False"""
```

Comment:
It is great to have a docstring, but when the test failed you cannot see it. 

There is something special with the test names: you never call them in code yourself so you can give them very long names. Just move the docstring to the test name.

```suggestion
def test_zero_is_not_power_of_2():
```